### PR TITLE
Pm/paper prototype

### PR DESCRIPTION
**Pull request by tobiasr97**
Created the paperprototype for the Boutique-Hotel-Technikum app. Please review!

### CODE REVIEW

File: specs/pm/~$ototype for Advanced Webtechnologies.docx
Comment (alexisschaffer) [inline specs/pm/~$ototype for Advanced Webtechnologies.docx:1]: I would put files begining with `~$` in the .gitignore file to remove those word temp files

File: specs/pm/~WRL1777.tmp
Comment (alexisschaffer) [inline specs/pm/~WRL1777.tmp:1]: Same for `.tmp` files

**Review by alexisschaffer — CHANGES_REQUESTED** [review]
To file types should probably be added to the .gitignore file please.

File: specs/pm/~$ototype for Advanced Webtechnologies.docx
Comment (tobiasr97) [reply specs/pm/~$ototype for Advanced Webtechnologies.docx:1]: Added to .gitignore

**Review by alexisschaffer — APPROVED** [review]
Thanks, that's all!

**Pull request merged**